modules/runners/variables.tf

```
@@ -761,3 +761,9 @@ variable "user_agent" {
   type        = string
   default     = null
 }
+
+variable "ec2_custom_allowed_tags_creation" {
```

suggest to renamen the parameter
```suggestion
variable "runner_custom_allowed_tags" {
```